config/i18n-tasks.yml

```diff
@@ -113,6 +113,7 @@ ignore_unused:
 # - 'simple_form.{placeholders,hints,labels}.*'
 # - 'simple_form.{error_notification,required}.:'
 - decidim.components.surveys.settings.global.enable_results
+- survey_results.questionnaire_show.see_results
```

Comment:
why this translations are here?

Reply:
check @NicoleSo13 's comment: https://github.com/CodiTramuntana/decidim-module-survey_results/pull/3#issuecomment-1532501638